### PR TITLE
NLog ElasticSearch Target

### PR DESCRIPTION
**Pull request by snakefoot**
NLog Target that exports directly to Elastic Cloud or individual Elasticsearch nodes

Docs Preview: https://github.com/snakefoot/ecs-dotnet/tree/main/src/Elastic.NLog.Targets

### CODE REVIEW

File: src/Elastic.NLog.Targets/README.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+## Packages
+
+The .NET assemblies are published to NuGet under the package name [Elastic.NLog.Targets](http://nuget.org/packages/Elastic.NLog.Targets)
+
+## How to use from API
+
+```csharp
+var config = new LoggingConfiguration();
+var elasticTarget = new ElasticsearchTarget("elastic") { Layout = new EcsLayout(), NodesUri = "http://localhost:9200" };
```
Comment (Mpdreamz) [inline src/Elastic.NLog.Targets/README.md:13]: Specifying the Layout is not necessary here correct? it defaults to this?

**Review by Mpdreamz — DISMISSED** [review]
LGTM in principle, what a great addition to datashippers!

Could you address the conflict on the `sln` file?

File: src/Elastic.NLog.Targets/README.md (hunk 1 above)
Comment (snakefoot) [reply src/Elastic.NLog.Targets/README.md:13]: It was mostly to show that the `ElasticsearchTarget` relies on the `EcsLayout`, and you can configure the `EcsLayout` if you want to override the default output.

**Comment by snakefoot** [comment]
> what a great addition to datashippers!

Yes was just missing a "Flush"-method on the shipper, as both NLog and log4net has the ability to "Flush" its targets/appenders.

> Could you address the conflict on the sln file?

Conflicts resolved

**Comment by Mpdreamz** [comment]
run docs-build

**Comment by snakefoot** [comment]
@Mpdreamz Hope I nailed the integration-test this time. Sorry about the retries.

**Comment by Mpdreamz** [comment]
run docs-build

**Comment by Mpdreamz** [comment]
> @Mpdreamz Hope I nailed the integration-test this time. Sorry about the retries.

Don't be! I'll gladly retry until we get there, looking forward to releasing this!

**Comment by Mpdreamz** [comment]
run docs-build

**Comment by snakefoot** [comment]
Finally green build :)

**Comment by Mpdreamz** [comment]
run docs-build

**Review by Mpdreamz — APPROVED** [review]
LGTM!

**Comment by Mpdreamz** [comment]
@snakefoot thanks for this new package!

I will be out starting today for 10 days but will release this as soon as I am back, after looking at the other open PR.

**Pull request merged**